app/src/main/report/components/ReportsList/ReportsList.ts

```diff
@@ -0,0 +1,43 @@
+import { connect } from 'react-redux';
+import { Dispatch } from "redux";
+import { compose, lifecycle, branch, renderComponent } from "recompose";
+import { createSelector } from "reselect";
+import withLifecycle from '@hocs/with-lifecycle';
+
+import {ReportAppState} from "../../reducers/reportAppReducers";
+import { reportsActions } from "../../actions/reportsActions";
+import {ReportsListComponent, ReportsListComponentProps} from "./ReportsListComponent";
+import { LoadingElement } from "../../../shared/partials/LoadingElement/LoadingElement";
+import {getDisplayedReportsListSelector} from "./reportsListSelectors";
+
+export interface ReportsListContainerProps extends ReportsListComponentProps {
+    getReports: () => void;
+    ready: boolean;
+}
+
+const lifecyleProps = {
+    onDidMount(props: ReportsListContainerProps) {
+        console.log('pr', props)
```

Comment:
This seems better to me now it has the `props` typing, thanks. Is this functionality under test?